### PR TITLE
added chrid

### PR DESCRIPTION
**Pull request by Yoav-Guterman**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+html,
+body {
+    display: chrid;
```
Comment (shaharsol) [inline assets/css/style.css]: typo, chrid should have been grid.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+}
```
Comment (shaharsol) [inline assets/css/style.css:5]: missing grid-template-column and grid-template-row

**Review by shaharsol — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (Yoav-Guterman) [reply assets/css/style.css]: done

File: assets/css/style.css (hunk 2 above)
Comment (Yoav-Guterman) [reply assets/css/style.css:5]: done

**Review by shaharsol — APPROVED** [review]

**Pull request merged**